testsuite/pom.xml

```diff
@@ -154,6 +154,10 @@
              See https://issues.apache.org/jira/browse/MNG-2496 -->
         <version.org.apache.xalan>2.7.3</version.org.apache.xalan>
 
+	<!-- This overrides default version specified in parent pom - reason is to enable TCP/IP communication,
+	     see https://maven.apache.org/surefire/maven-surefire-plugin/examples/process-communication.html -->
+        <version.surefire.plugin>3.1.2</version.surefire.plugin>
```

Comment:
I think this should be in the root pom, as surefire is used outside the testsuite tree, and we'll likely lose track of this separate setting here.

The xalan version above should perhaps move to the root pom too (I'm not asking you to do that; I'm just explaining what may look like an inconsistency.) But it's a somewhat different case, as there the dependency is only used in the testsuite module. I think part of the idea with xalan was to keep the property out of places where people could misuse it to configure production code versions.

Reply:
@bstansberry thank you for your review! I've moved the version definition to the root pom as you suggested. I also included `forkNode` definition in the surefire plugin configuration there too.

Let me know if you see some more issues, thanks! :slightly_smiling_face: